### PR TITLE
First draft of new blog post

### PR DESCRIPTION
**Pull request by TrechNex**
Preview link: https://deploy-preview-102--competent-wright-57cc1e.netlify.app/posts/first-steps-towards-the-future/

### CODE REVIEW

**Comment by TrechNex** [comment]
The bullet points do work on `hugo server -D`, but apparently not in the preview deployment. That is... strange.

File: content/posts/october-2020-update.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+You can see full details about how that progressed via this Github Issue: [#477](https://github.com/glimpse-editor/Glimpse/issues/477)
+
+## Glimpse Image Editor 0.2.0 released as a Linux AppImage
+As promised, on 2020-09-20 we released our very first automatically-built Linux AppImage. This "universal binary" is built from the tip of our development branch, and is primarily intended for Linux enthusiasts and testing purposes.
+
+You can see full details about how that progressed via this Github Issue: [#108](https://github.com/glimpse-editor/Glimpse/issues/108)
+
+## Progress on distributing our Windows port via Scoop
+As you will see on our downloads page, you can already download Glimpse Image Editor via [Chocolatey](https://chocolatey.org/packages/glimpse/) and [WinGet](https://winget.run/pkg/Glimpse/Glimpse). We understand that [bb010g](https://github.com/bb010g) is currently working on publishing our Windows installer for the automated [Scoop](https://scoop.sh/) package manager.
```
Comment (BrainBlasted) [inline content/posts/october-2020-update.md:32]: CC: @bb010g are you okay with your profile being shared in this blog post?

File: content/posts/october-2020-update.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: "October 2020 Update"
```
Comment (TrechNex) [inline content/posts/october-2020-update.md:2]: I'm open to ideas on a more interesting title. There's a lot of things that went on, so it was tricky to come up with a "theme"

File: content/posts/october-2020-update.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Planning for the future with Glimpse Redux
+Bobby Moss has already created an initial private fork of the pre-release code for the GNU Image Manipulation Program 3.0.0. Over the coming year he will gradually customize that fork, and once it is at least "alpha" quality that work will be pushed to a new branch in the [existing Github repository](https://github.com/glimpse-editor/Glimpse) we use to host the Glimpse source code.
+
+We believe this work will be crucial in guaranteeing the future of the Glimpse project, as the GNU Image Manipulation Program developers will eventually switch across to their future 3.0.0 technology stack and deprecate 2.10.x.
+
+## Early investigation work starts on Glimpse NX
+Christopher Davis has already started familiarising himself with the technology stack we plan to use for this project. There have also been some excellent ideas introduced by our community on the Glimpse Discord server. We encourage anyone that is interested in participating with this effort to consider [requesting to join the server](https://discord.gg/hZhRceq).
```
Comment (BrainBlasted) [inline content/posts/october-2020-update.md:58]:
```suggestion
Work has begun preparing the tech stack for Glimpse NX. Thanks to work from GNOME contributor [Bilal Elmoussaoui](https://github.com/bilelmoussaoui) and our own Christopher Davis, the GTK4-rs bindings have been receiving updates as the final GTK4 release approaches. As part of familiarizing himself with the new APIs and new technologies behind GTK4, Christopher has ported various third-party GNOME apps to GTK4.

In addition to the GTK4 bindings, Bilal was able to generate working [Babl and Gegl bindings](https://gitlab.gnome.org/bilelmoussaoui/gegl-rs). Bilal is an amazing developer, and his work on bindings is incredibly helpful.

Alongside the technological efforts, design work on the next version of Glimpse has begun in earnest on the [Glimpse NX design repository](https://github.com/glimpse-editor/glimpse-nx-design). If you wish to contribute to these discussions and have a part in shaping the future of Glimpse, please read our guidelines and [request to join the server](https://discord.gg/hZhRceq).
```

**Review by BrainBlasted — CHANGES_REQUESTED** [review]

File: content/posts/october-2020-update.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Planning for the future with Glimpse Redux
+Bobby Moss has already created an initial private fork of the pre-release code for the GNU Image Manipulation Program 3.0.0. Over the coming year he will gradually customize that fork, and once it is at least "alpha" quality that work will be pushed to a new branch in the [existing Github repository](https://github.com/glimpse-editor/Glimpse) we use to host the Glimpse source code.
+
+We believe this work will be crucial in guaranteeing the future of the Glimpse project, as the GNU Image Manipulation Program developers will eventually switch across to their future 3.0.0 technology stack and deprecate 2.10.x.
+
+## Early investigation work starts on Glimpse NX
+Work has begun preparing the tech stack for Glimpse NX. Thanks to work from GNOME contributor [Bilal Elmoussaoui](https://github.com/bilelmoussaoui) and our own Christopher Davis, the GTK4-rs bindings have been receiving updates as the final GTK4 release approaches. As part of familiarizing himself with the new APIs and new technologies behind GTK4, Christopher has ported various third-party GNOME apps to GTK4.
```
Comment (BrainBlasted) [inline content/posts/october-2020-update.md:58]:
```suggestion
Work has begun preparing the tech stack for Glimpse NX. Thanks to work from GNOME contributor [Bilal Elmoussaoui](https://github.com/bilelmoussaoui) and our own Christopher Davis, the GTK4-rs bindings have been receiving updates as the final GTK4 release approaches. As part of familiarizing himself with the new APIs and new technologies behind GTK4, Christopher has ported various third-party GNOME apps to GTK4. While porting he uncovered different issues and gaps that the upstream GTK team are investigating:

* [Notifications don't activate windows on wayland](https://gitlab.gnome.org/GNOME/gtk/-/issues/3261)
* [Opening popover causes noticeable slowdown](https://gitlab.gnome.org/GNOME/gtk/-/issues/3264)
* [Cannot open popovers with touch-only input](https://gitlab.gnome.org/GNOME/gtk/-/issues/3279)
* [Provide a way to make custom popover menus matching MenuModel styling](https://gitlab.gnome.org/GNOME/gtk/-/issues/3260)
```

**Review by BrainBlasted — CHANGES_REQUESTED** [review]

File: content/posts/october-2020-update.md (hunk 1 above)
Comment (bb010g) [reply content/posts/october-2020-update.md:33]: Yes.

File: content/posts/october-2020-update.md (hunk 2 above)
Comment (TrechNex) [reply content/posts/october-2020-update.md]: Picked up @BrainBlasted idea "First Steps Towards The Future@

**Pull request merged**